.opencode/commands/career-ops-latex.md

```
@@ -0,0 +1,10 @@
+---
+description: Export CV as LaTeX/Overleaf .tex
+---
+
+Export a tailored, ATS-optimized CV as a LaTeX `.tex` file (and optionally compile to PDF) using career-ops latex mode.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)